NoteSalihMVVM/Scenes/Auth/Login/LoginViewController.swift

```diff
@@ -6,11 +6,135 @@
 //
 
 import UIKit
+import MobilliumBuilders
+import TinyConstraints
+import UIComponents
 
 final class LoginViewController: BaseViewController<LoginViewModel> {
     
+    private let scrollView = UIScrollViewBuilder().build()
+    private let contentView = UIViewBuilder().build()
+    private let header: AuthHeaderView = {
+        let header = AuthHeaderView()
+        return header
+    }()
+    
+    private let formStackView = UIStackViewBuilder()
+        .axis(.vertical)
+        .alignment(.fill)
+        .distribution(.fillEqually)
+        .spacing(14)
+        .build()
+    private let emailTextField: AuthTextField = {
+        let textField = AuthTextField()
+        return textField
+    }()
+    private let passwordTextField: AuthTextField = {
+        let textField = AuthTextField()
+        textField.isSecureTextEntry = true
+        return textField
+    }()
+    
+    private let buttonStackView = UIStackViewBuilder()
+        .axis(.vertical)
+        .alignment(.trailing)
+        .distribution(.fill)
+        .spacing(27)
+        .build()
+    private let forgotPasswordButton = UIButtonBuilder()
+        .titleFont(.font(.josefinSansRegular, size: 13))
+        .titleColor(.appEbonyClay)
+        .build()
+    private let loginButton: AuthButton = {
+        let button = AuthButton()
+        return button
+    }()
+    
+    private let footer: AuthFooterView = {
```

Comment:
```suggestion
    private let footerView = AuthFooterView()
```
direk init edilebilir değişken kullanlmamış ayrıca isimlendirmede view ise belirtilmeli

Reply:
düzeltilmeli

Reply:
fix